examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>First sentence here again repeating yourself 😅</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Kind of conflicted here :) </p><p> </p><p>I am trying to not break the consistency of the other notebooks (such as https://github.com/huggingface/notebooks/blob/main/examples/image_classification.ipynb) in terms of how content is outlined. </p><p> </p><p>But I do see your point. </p>

Reply:
Ah I see...its just my opinion. feel free to stick with it if it's following a pattern you like

Reply:
And I second you. The latest updates I am about to push will get rid of it :)